flink-connector-debezium/src/main/java/com/alibaba/ververica/cdc/debezium/table/RowDataDebeziumDeserializeSchema.java

```diff
@@ -85,11 +85,17 @@
 	 */
 	private final ValueValidator validator;
 
-	public RowDataDebeziumDeserializeSchema(RowType rowType, TypeInformation<RowData> resultTypeInfo, ValueValidator validator, ZoneId serverTimeZone) {
+	/**
+	 * whether to capture the updates if that have no change or not.
+	 */
+	private final boolean captureUnchangedUpdates;
+
+	public RowDataDebeziumDeserializeSchema(RowType rowType, TypeInformation<RowData> resultTypeInfo, ValueValidator validator, ZoneId serverTimeZone, boolean captureUnchangedUpdates) {
```

Comment:
The constructor is getting longger. Could you make the constructor private and provde a builder for this class?

Reply:
> The constructor is getting longger. Could you make the constructor private and provide a builder for this class?

Okay, the construction method of MySQLTableSource is too long, do I need to make it a builder?